local/src/client/verifier-client.cc

```diff
@@ -276,6 +276,27 @@ ClientReplayFileHandler::ssn_open(YAML::Node const &node)
         _ssn->is_h3 = true;
       }
     }
+    // parse the proxy protocol node
+    auto const pp_node = parse_for_protocol_node(protocol_sequence_node, YAML_SSN_PROTOCOL_PP_NAME);
+    if (!pp_node.is_ok()) {
+      errata.note(std::move(pp_node.errata()));
+      return errata;
+    }
+    if (pp_node.result().IsDefined()) {
+      auto const &pp_version_node = pp_node.result()[YAML_SSN_PROTOCOL_VERSION];
+      if (pp_version_node.IsDefined() &&
+          (pp_version_node.Scalar() == "1" || pp_version_node.Scalar() == "2"))
+      {
+        errata.note(S_DIAG, "Enabling PROXY protocol for this session.");
+        // set proxy protocol header version as specified
+        _ssn->pp_version =
+            (pp_version_node.Scalar() == "1") ? ProxyProtocolVersion::V1 : ProxyProtocolVersion::V2;
+      } else {
+        // unspecified or invalid versions
+        errata.note(S_ERROR, "Invalid PROXY protocol version specified.");
```

Comment:
I have enough scar tissue to know that it will be helpful to know the path and line number for this. See the example on line 308 below.